### PR TITLE
Updated to work with Wagtail 2.7

### PR DESCRIPTION
**Pull request by Hopiu**

### CODE REVIEW

File: wagtail_modeltranslation/static/wagtail_modeltranslation/js/copy_stream_fields.js [hunk 1]
```diff
@@ -7,27 +7,47 @@ $(document).ready(function(){
...
+    var fieldName = "";
+    if(versionCompare(WAGTAIL_VERSION,'2.6.0', {zeroExtend: true})===-1){
+      // Wagtail < 2.6 
       header = $(currentStreamField).children('h2')[0];
+      //Search for the input field so that we can get is id to know the field's name.
+      let streamFieldDiv = $(currentStreamField).find('div.sequence-container.sequence-type-stream')[0];
+      let fieldInfos = $(streamFieldDiv).find('input')[0].id.split('-')[0];
+      let lastUnderscore = fieldInfos.lastIndexOf("_");
+      fieldName = fieldInfos.substring(0, lastUnderscore);
+      fieldLang = fieldInfos.substring(lastUnderscore + 1, fieldInfos.length);
```
Comment (DiogoMarques29) [inline wagtail_modeltranslation/static/wagtail_modeltranslation/js/copy_stream_fields.js]: These 4 lines are repeated in every section. Can you please move it outside?

File: wagtail_modeltranslation/static/wagtail_modeltranslation/js/version_compare.js [hunk 2]
```diff
@@ -8,20 +8,20 @@
...
  *     <li>
- *         <tt>lexicographical: true</tt> compares each part of the version strings lexicographically instead of
+ *         <tt>lexicographical: false</tt> compares each part of the version strings lexicographically instead of
  *         naturally; this allows suffixes such as "b" or "dev" but will cause "1.10" to be considered smaller than
  *         "1.2".
  *     </li>
  *     <li>
- *         <tt>zeroExtend: true</tt> changes the result if one version string has less parts than the other. In
+ *         <tt>zeroExtend: false</tt> changes the result if one version string has less parts than the other. In
  *         this case the shorter string will be padded with "zero" parts instead of being considered smaller.
```
Comment (DiogoMarques29) [inline wagtail_modeltranslation/static/wagtail_modeltranslation/js/version_compare.js]: Can you explain this?

**Comment by DiogoMarques29** [comment]
Hi @Hopiu 
Thank you for the PR.
I noticed that you started using ES6, with `let` and template strings. We would like to maintain backwards compatibility for old browsers, so if you could not use ES6 we would appreciate it.

**Comment by Hopiu** [comment]
o.k. i'll rewrite it to support older browsers. Is IE still supported by
wagtail admin?

**Comment by DiogoMarques29** [comment]
We think older versions of wagtail still support it. We are still trying to supporting wagtail >= 1.12 🤞

File: wagtail_modeltranslation/static/wagtail_modeltranslation/js/version_compare.js (hunk 2 above)
Comment (Hopiu) [reply wagtail_modeltranslation/static/wagtail_modeltranslation/js/version_compare.js]: Sorry, this was wrong. I mixed it up with the default values of the parameter. I corrected and clarified the js doc in the newest commit.

Actually I'm unsure, whether this comparison should happen in js at all. Probably one should create different js files for different Wagtail versions and do the switching in `wagtail_hooks.py`. What do you think?

**Comment by urlsangel** [comment]
Hi @DiogoMarques29  and @Hopiu 

Firstly, thanks for all your hard work on this package - it's a real help for localisation of Wagtail sites.

We're in the process of upgrading a site to Wagtail `2.7.1`, and have seen that the major StreamField update from `2.7` causes some JS errors with `wagtail-modeltranslation`  - this PR looks like it fixes that?

If that's the case, do you know when this PR might be merged and released?

Thanks!

Russell.

**Comment by DiogoMarques29** [comment]
Closed in favour of a6bfed841dc482f1ad5139d153449f65bc13d51f

**Pull request closed without merging**